lib/codecs/Cargo.toml

```diff
@@ -45,6 +46,7 @@ similar-asserts = "1.5.0"
 vector-core = { path = "../vector-core", default-features = false, features = ["vrl", "test"] }
 uuid = { version = "1", default-features = false, features = ["serde", "v4"] }
 rstest = "0.21.0"
+tracing-test = "0.2.5"
```

Comment:
Found this and it was useful for asserting that tracings logs were produced

I see that this dependency was used in previous versions, such as in vector `0.28.0`, but couldn't find it in the current version. 
Are you okay with using this? The license is APACHE 2.0 OR MIT (https://github.com/dbrgn/tracing-test)

Reply:
I think it should be added to the licenses csv anyways

Reply:
👍 you can regenerate them by installing https://github.com/DataDog/rust-license-tool and running `cargo vdev build licenses`.

Reply:
<img width="794" alt="image" src="https://github.com/user-attachments/assets/8e9b6b06-6220-408a-9a9f-a2ee44933ec8">
<img width="669" alt="image" src="https://github.com/user-attachments/assets/92dc4bd6-1b15-4b22-b4ad-e62a3eab537c">

Reply:
<img width="783" alt="image" src="https://github.com/user-attachments/assets/c092a632-adc3-481d-8e26-560cb29b2741">
the command does nothing but touching the `Cargo.lock`
<img width="709" alt="image" src="https://github.com/user-attachments/assets/858ecb1f-fb4a-4ffb-b1f5-7161c0defd0b">

I don't know if I'm doing anything wrong...
The dependency is added in the `lib/codecs/Cargo.toml`, not in the workspace root `Cargo.toml`, is `dd-rust-license-tool` aware of dependencies across workspace crates?

Reply:
Ah, you'll want to run: `cargo vdev build licenses` (or `dd-rust-license-tool write`).

Reply:
yes, see the second command of the first screenshot! I ran `cargo vdev build licenses` but didn't change any file/output anything, altough dd-rust-license-tool is installed

Reply:
Oh I see, yes, you are right. It seems like `dd-rust-license-tool` might not be picking up dependencies of subcrates. 🤔

Reply:
Oh, actually, I think this is because this is a dev-dependency rather than a dependency that is included in the distributed artifacts.

Reply:
oh, okay!! everything alright then :)

(I'll answer the other comments asap, sorry I had not time these days)